.github/workflows/rsc/dockerfile_lint_rules.yaml

```diff
@@ -0,0 +1,215 @@
+---
+profile:
+  name: "Default"
+  description: "Default Profile. Checks basic syntax."
+  includes:
+# recommended_label_rules.yaml
```

Comment:
should this line be removed?

Reply:
I did the minimal changes to the original: https://github.com/projectatomic/dockerfile_lint/blob/master/config/default_rules.yaml

E.g. just removed the last few lines of required commands, and did some whitespace cleanup that was caught by statick.  I didn't think further cleanup was really warranted, though I'm not opposed to removing that line.

Reply:
i think we should remove it since the referenced file doesn't exist in this repo, nor does it exist in the original repo.

Reply:
f36bcd4